### DISCUSSION TITLE
*Newbie* Github pages not applying CSS to my page

### DISCUSSION THREAD

**Discussion by edmundsamina** [community/community#31401] [New to GitHub] [answered]
Hello, I am a newbie trying to make a tribute page, but GitHub is not applying the CSS to my page. The code worked fine in codepen and loaded fine. I have looked up the solutions and tried a few but cannot figure out what I have done wrong. Any help would be much appreciated. 

Here is my repository: 
https://github.com/edmundsamina/tribute_page.git

Here is the website link:
https://edmundsamina.github.io/tribute_page/

Many Thanks!

**Comment by airtower-luna** [comment, marked as answer]
The problems (two) [are here](https://github.com/edmundsamina/tribute_page/blob/91e354422dbb66eaf03d256befc702afd5dee15f/index.html#L4):

```html
        <link rel=“stylesheet”  href="css/style.css">
```

1. Your stylesheet isn't in a `css/` subdirectory in the repository, it's right next to the `index.html` file. Remove the directory from the `href` value.
2. The `rel` attribute must use simple quotes (`""`) not paired ones (`“”`) around the value.

**Reply by edmundsamina** [reply]
Thank you so much, I don't know how I didn't spot that! You are amazing
